web_poet/page_inputs/stats.py

```diff
@@ -0,0 +1,17 @@
+from abc import abstractmethod
+from typing import Any
+
+from web_poet.pages import Injectable
+
+
+class Stats(Injectable):
```

Comment:
This is the only page input that inherits from `Injectable`. I wonder if I am missing some potential issue by doing that.

Reply:
This would cause problems in `deserialize()` but you already have specoial handling for this class there.

Maybe it could cause similar problems in other places that use `andi` and need to determine which deps are leaf deps?

Reply:
Why do you need to make it Injectable? Would it make sense to add a comment about it somewhere in the code?